config/labels/analyzers/clang-tidy.json

```diff
@@ -4798,6 +4847,12 @@
       "profile:extreme",
       "severity:LOW"
     ],
+    "cppcoreguidelines-rvalue-reference-param-not-moved": [
```

Comment:
This should not be in the default profile, because if only parts of the object are moved but not the whole object, then by default, the checker would warn. I don't have results to support my claim, but I think this could lead to annoying warnings.

I would leave this out of the default profile for now.